gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerKotlinSupportPlugin.kt

```diff
@@ -6,23 +6,48 @@
 package org.jetbrains.compose
 
 import org.gradle.api.provider.Provider
+import org.jetbrains.compose.internal.webExt
 import org.jetbrains.kotlin.gradle.plugin.*
+import org.jetbrains.kotlin.gradle.targets.js.ir.KotlinJsIrTarget
 
 class ComposeCompilerKotlinSupportPlugin : KotlinCompilerPluginSupportPlugin {
     override fun getCompilerPluginId(): String =
-        "org.jetbrains.compose"
+        "androidx.compose.compiler.plugins.kotlin"
```

Comment:
Why this change?

Reply:
`getCompilerPluginId` is used by Kotlin Gradle plugin, when building CLI args for Kotlin compiler.
Basically it transforms options, returned by `applyToCompilation`, to `plugin:$compilerId:${option.key}=${option.value}`
The compiler plugin id is defined in https://github.com/androidx/androidx/blob/androidx-main/compose/compiler/compiler-hosted/src/main/java/androidx/compose/compiler/plugins/kotlin/ComposePlugin.kt#L52

Our build of the Compose compiler plugin does not override the plugin id.
Previous value was incorrect.